lib/chef/knife/bootstrap.rb

```diff
@@ -432,6 +433,22 @@ def validate_options!
         true
       end
 
+      # Ensure options are valid by checking bootstrap version values.
+      #
+      # The method call will cause the program to exit(1) if:
+      #   * Bootstrap version is greater than or equal to 15 for chef 14 or lower
+      #   * Pre-release options is passed for chef 14 or lower
+      #
+      # @return [TrueClass] If options are valid.
+      def validate_bootstrap_version_options!
+        if Chef::VERSION.to_i < 15 && bootstrap_version_gte_15?
```

Comment:
There's no need for the Chef::VERSION check since we know we're on 14

Reply:
Yes, @tas50 there is no need of checking version but for sanity purchase, we may think of?